README.md

```diff
@@ -11,6 +11,103 @@ go get github.com/moolitayer/golang/glog # Dependency
 go get github.com/moolitayer/awx-client-go/awx
 ```
 
+## Usage
+### import
+```
+import 	"github.com/moolitayer/awx-client-go/awx"
+```
+
+### Creating a connection:
+```
+# Uses the builder pattern:
+connection, err := awx.NewConnectionBuilder().
+  Url("http://awx.example.com/api").
+  Username(username).
+  Password(password).
+  Token("TOKEN").
+  Bearer("BEARER").
+  CAFile("/etc/pki/tls/cert.pem").
+  Insecure(insecure).
```

Comment:
It'd probably be a good idea to mention which of these are optional and which are mandatory.